### PR TITLE
Typo and grammatical correction in French ASVS V4

### PR DESCRIPTION
**Pull request by Aif4thah**
Hi,

I propose a PR directly as it involves just 4 grammatical corrections in 2 files in the ASVS v4 (French version).

Regards

### CODE REVIEW

**Comment by tghosth** [comment]
Can you confirm @clallier94 @inaz0 @SPoint42 @Marx314?

**Comment by inaz0** [comment]
It's OK for me.

File: 4.0/fr/0x10-V1-Architecture.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Les exigences architecturales sont intrinsèques au code, et donc difficiles à
 
 | # | Description | L1 | L2 | L3 | CWE |
 | :---: | :--- | :---: | :---:| :---: | :---: |
-| **1.7.1** | Vérifier qu'un format de journalisation communs soit utilisés dans le système.  ([C9](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | 1009 |
+| **1.7.1** | Vérifier qu'un format de journalisation commun soit utilisé dans le système.  ([C9](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | 1009 |
```
Comment (SPoint42) [inline 4.0/fr/0x10-V1-Architecture.md:80]: approve

File: 4.0/fr/0x11-V2-Authentication.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Les fournisseurs de services d'accréditation (CSP) fournissent une identité f
...
 | **2.1.8** | Vérifiez qu'un compteur de force de mot de passe est fourni pour aider les utilisateurs à définir un mot de passe plus fort. | ✓ | ✓ | ✓ | 521 | 5.1.1.2 |
 | **2.1.9** | Vérifiez qu'il n'existe pas de règles de composition des mots de passe limitant le type de caractères autorisés. Il ne doit pas y avoir restrictions sur l'utilisation de majuscules ou de minuscules, de chiffres ou de caractères spéciaux. ([C6](https://owasp.org/www-project-proactive-controls/#div-numbering)) | ✓ | ✓ | ✓ | 521 | 5.1.1.2 |
-| **2.1.10** | Vérifiez qu'il n'y a pas d'exigences en matière de rotation périodique du mots de passe ou d'historique des mots de passe. | ✓ | ✓ | ✓ | 263 | 5.1.1.2 |
+| **2.1.10** | Vérifiez qu'il n'y a pas d'exigences en matière de rotation périodique du mot de passe ou d'historique des mots de passe. | ✓ | ✓ | ✓ | 263 | 5.1.1.2 |
```
Comment (SPoint42) [inline 4.0/fr/0x11-V2-Authentication.md:52]: Approve

File: 4.0/fr/0x11-V2-Authentication.md [hunk 3]
```diff
@@ -63,12 +63,12 @@ Le NIST considère le courrier électronique et les SMS comme des [types d'authe
...
+| **2.2.1** | Vérifiez que les contrôles anti-automatisation sont efficaces pour atténuer les attaques par violation des tests d'accréditation, par énumération exhaustive (brute-force) et par verrouillage de compte. Ces contrôles comprennent le blocage des mots de passe les plus courants, les verrouillages progressifs, la limitation de débit, les CAPTCHA, les délais toujours plus longs entre les tentatives, les restrictions d'adresse IP ou les restrictions basées sur le risque tel que l'emplacement, la première connexion sur un appareil, les tentatives récentes de déverrouillage du compte, ou autres. Vérifiez qu'il n'y a pas plus de 100 tentatives infructueuses par heure sur un seul compte. | ✓ | ✓ | ✓ | 307 | 5.2.2 / 5.1.1.2 / 5.1.4.2 / 5.1.5.2 |
```
Comment (SPoint42) [inline 4.0/fr/0x11-V2-Authentication.md:66]: Approve

File: 4.0/fr/0x11-V2-Authentication.md [hunk 4]
```diff
@@ -63,12 +63,12 @@ Le NIST considère le courrier électronique et les SMS comme des [types d'authe
...
 | **2.2.4** | Vérifier la résistance à l'usurpation d'identité contre le phishing, comme l'utilisation de l'authentification multi-facteurs, les dispositifs cryptographiques avec intention (comme les clés connectées avec un "push to authenticate"), ou à des niveaux AAL supérieurs, les certificats côté client. | | | ✓ | 308 | 5.2.5 |
 | **2.2.5** | Vérifier que lorsqu'un fournisseur de services d'accréditation (CSP) et l'application vérifiant l'authentification sont séparés, un TLS mutuellement authentifié est en place entre les deux points terminaux. | | | ✓ | 319 | 5.2.6 |
-| **2.2.6** | Vérifier la résistance au attaque de rejeu par l'utilisation obligatoire de dispositifs OTP, d'authentificateurs cryptographiques ou de codes de consultation. | | | ✓ | 308 | 5.2.8 |
+| **2.2.6** | Vérifier la résistance aux attaques de rejeu par l'utilisation obligatoire de dispositifs OTP, d'authentificateurs cryptographiques ou de codes de consultation. | | | ✓ | 308 | 5.2.8 |
```
Comment (SPoint42) [inline 4.0/fr/0x11-V2-Authentication.md:71]: Approve

**Review by SPoint42 — APPROVED** [review]

**Review by SPoint42 — APPROVED** [review]

**Review by inaz0 — APPROVED** [review]

**Comment by tghosth** [comment]
Thanks @SPoint42 @inaz0 !

**Review by tghosth — APPROVED** [review]

**Comment by tghosth** [comment]
And thanks @Aif4thah!

**Pull request merged**